pkg/operation/botanist/component/vpnseedserver/monitoring.go

```diff
@@ -76,6 +77,12 @@ const (
 	monitoringMetricEnvoyClusterUpstreamCxLengthMsSum                          = "envoy_cluster_upstream_cx_length_ms_sum"
 	monitoringMetricEnvoyHttpDownstreamCxLengthMsBucket                        = "envoy_http_downstream_cx_length_ms_bucket"
 	monitoringMetricEnvoyHttpDownstreamCxLengthMsSum                           = "envoy_http_downstream_cx_length_ms_sum"
+
+	monitoringMetricOpenvpnServerClientReceivedBytesTotal      = "openvpn_server_client_received_bytes_total"
```

Comment:
just for reference, the definition of the metric in teh openvpn_exporter: https://github.com/kumina/openvpn_exporter/blob/9d468c00154aa5b3f921573e2b0354b6a7c5699f/exporters/openvpn_exporter.go#L116

the `ignoreIndividuals` feature can be used to control the details (like the real_address label) and hence the cardinality.
https://github.com/kumina/openvpn_exporter/blob/9d468c00154aa5b3f921573e2b0354b6a7c5699f/exporters/openvpn_exporter.go#L97-L103

https://github.com/kumina/openvpn_exporter/blob/9d468c00154aa5b3f921573e2b0354b6a7c5699f/main.go#L31